RecoLocalTracker/SiPixelRecHits/python/PixelCPEGeneric_cfi.py

```diff
@@ -55,3 +54,12 @@
 (phase2_tracker & (phase2_squarePixels | phase2_3DPixels)).toModify(PixelCPEGenericESProducer,
                                                                     NoTemplateErrorsWhenNoTrkAngles = False # use genErrors in the seeding step (when no track angles are available)
                                                                     )
+# customize phase2 CPE generic for bricked pixels
+from Configuration.Eras.Modifier_phase2_brickedPixels_cff import phase2_brickedPixels
+from Configuration.ProcessModifiers.PixelCPEGenericForBricked_cff import PixelCPEGenericForBricked
+(phase2_tracker & (phase2_3DPixels & phase2_brickedPixels & PixelCPEGenericForBricked)).toModify(PixelCPEGenericESProducer,
```

Comment:
why does the `phase2_3DPixels` modifier need to be here? 
I understand that T27 is an hybrid geometry but don't the modifications needed to run bricked and 3D factorize?

Reply:
right!

Reply:
This was corrected in the last commit.